conf/modules.config

```diff
@@ -426,7 +359,7 @@ process {
     if (params.polishing_on) {
         withName: LONGRANGER_MKREF {
             if(System.getenv('GITHUB_ACTION') != null ) {
```

Comment:
This works but like I commented last time, maybe more correctly to override the default container to a different one in the github_test config profile.